### PR TITLE
Update wordlist update workflow with new events

### PR DESCRIPTION
**Pull request by irina060981**
For the issue https://github.com/alpheios-project/alpheios-core/issues/398

- fix bug with creating unuseful actions for wordlist update

For the issue https://github.com/alpheios-project/alpheios-core/issues/201

- fix bug with updating currentSession, frequency and updateDT for wordlist update

### CODE REVIEW

File: packages/components/src/lib/controllers/ui-controller.js [hunk 1]
```diff
@@ -1665,9 +1665,6 @@ If no URLS are provided, will reset grammar data.
   onHomonymReady (homonym, source) {
     // if we have a request for updating wordlist for download then we need this event,
```
Comment (balmas) [inline packages/components/src/lib/controllers/ui-controller.js:1666]: we can remove the comment too, right?

File: packages/wordlist/src/controllers/wordlist-controller.js [hunk 2]
```diff
@@ -155,6 +159,21 @@ export default class WordlistController {
     WordlistController.evt.WORDLIST_UPDATED.pub(this.wordLists)
   }
 
+  /**
+   * Responds to a WORDLIST_UPDATE_HOMONYM_READY event by updating a wordlist item for a retrieved Homonym
+   * @param {Homonym} data
+   * Emits WORDITEM_UPDATED and WORDLIST_UPDATED events
+   */
+  onHomonymReadyForWordlistUpdate (data) {
+    console.info('onHomonymReadyForWordlistUpdate', data)
```
Comment (balmas) [inline packages/wordlist/src/controllers/wordlist-controller.js:168]: remove debugging print

File: packages/components/src/lib/controllers/ui-controller.js (hunk 1 above)
Comment (irina060981) [reply packages/components/src/lib/controllers/ui-controller.js:1666]: yes - removed

File: packages/wordlist/src/controllers/wordlist-controller.js (hunk 2 above)
Comment (irina060981) [reply packages/wordlist/src/controllers/wordlist-controller.js:168]: removed

**Pull request merged**